homeassistant/components/emoncms/__init__.py

```diff
@@ -1 +1,38 @@
 """The emoncms component."""
+
+from pyemoncms import EmoncmsClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_URL, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .coordinator import EmoncmsCoordinator
+
+PLATFORMS: list[str] = [Platform.SENSOR]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
Let's extend the ConfigEntry type to type the runtime_data_field

Reply:
done, if my understanding is correct...